backend/src/main/resources/templates/admin/zzolbot.html

```
@@ -0,0 +1,437 @@
+        <!-- 챗 영역 -->
+        <div>
+            <div class="chat-box" id="chatBox">
+                <div class="chat-empty" id="emptyHint">
+                    joinCode를 포함해 질문해보세요.<br>
+                    예: "A4BX 방에서 뭔가 잘못됐어", "방금 에러 났는데 A4BX 확인해줘"
+                </div>
+            </div>
+            <div class="input-row">
+                <input type="text" id="questionInput" placeholder="질문을 입력하세요..."
+                       autocomplete="off" autofocus/>
+                <button class="btn-send" id="sendBtn" onclick="sendQuestion()">전송</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)